gatsby/src/pages/borderline-substances/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { useStaticQuery } from "gatsby";
+
+import { mockBorderlineSubstancesPagesQueryData } from "@/hooks/useBorderlineSubstancesPages.test";
+
+import BorderlineSubstanceIntroductionPage, {
+	type BorderlineSubstanceIntroductionPageProps,
+} from "./";
+
+const props: BorderlineSubstanceIntroductionPageProps = {
+	data: {
+		bnfBorderlineSubstancesIntroduction: {
+			title: "Borderline Substances",
+			sections: [],
+		},
+	},
+};
+
+describe("BorderlineSubstanceIntroductionPage", () => {
+	beforeEach(() => {
+		(useStaticQuery as jest.Mock).mockReturnValue(
+			mockBorderlineSubstancesPagesQueryData
+		);
+	});
+
+	it("should match snapshot for page contents", () => {
+		render(<BorderlineSubstanceIntroductionPage {...props} />);
```

Comment:
Can you change the test to:
```
const { container } = render(<BorderlineSubstanceIntroductionPage {...props} />);
expect(container).toMatchSnapshot();
```

Just because it means the test is then self-contained and not affected by the layout component, related to some work in a seaprate PR